docs/08-guides.md

```diff
@@ -189,6 +189,44 @@ import { repeat } from "/web_modules/lit-html/directives/repeat.js";
 // ✔ snowpack installed: lit-html, lit-element [0.25s]
 ```
 
+### Svelvet
+
+[Svelvet](https://github.com/jakedeichert/svelvet) is a cli [svelte](https://github.com/sveltejs/svelte) compiler and file watcher that integrates directly with snowpack.
```

Comment:
this looks great! Since the rest of this docs site is all about how you now need to write apps that can be shipped directly to the browser, lets call out explicitly here that Svelvet compiles your source files so that you can write Svelte component syntax, use import URLs  that aren't fully resolved, etc.

Reply:
Updated the wording! Not sure if it's good enough yet... let me know if i should make more changes or if you want to, please go ahead! :)